### PR TITLE
fix: random topk rest port

### PR DESCRIPTION
**Pull request by bhavsarpratik**

### CODE REVIEW

File: tests/integration/issues/github_976/test_topk_rest_api.py [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+import pytest
+import random
 from urllib import request
 
 from jina.flow import Flow
 
 
 TOP_K = 2
-
+PORT = random.randint(1, 10000)
```
Comment (bwanglzu) [inline tests/integration/issues/github_976/test_topk_rest_api.py:10]: we have a random port method built-in codebase 
```
from jina.helper import random_port
```

**Review by bwanglzu — CHANGES_REQUESTED** [review]

**Comment by bwanglzu** [comment]
It looks like an integration test thing how do you think @JoanFM

File: tests/integration/issues/github_976/test_topk_rest_api.py (hunk 1 above)
Comment (hanxiao) [reply tests/integration/issues/github_976/test_topk_rest_api.py:10]: yes, the reason of not randomint is many ports below 40000 are reserved by OS already. Example: 22, 80

**Comment by bwanglzu** [comment]
hi @bhavsarpratik sorry I didn't realise that we're using `urllib.Request` here, do you think it's good to replace it with `requests`?  it's more readable and in our dependencies. Besides it's always better to respect our import logic

```
# python default
import json
import random
# community
import pytest
import requests
# Internal components
from jina.flow import Flow
from jina.helper import random_port
```
It's not necessary :) the rest lgtm!

**Comment by JoanFM** [comment]
> hi @bhavsarpratik sorry I didn't realise that we're using `urllib.Request` here, do you think it's good to replace it with `requests`? it's more readable and in our dependencies. Besides it's always better to respect our import logic
> 
> ```
> # python default
> import json
> import random
> # community
> import pytest
> import requests
> # Internal components
> from jina.flow import Flow
> from jina.helper import random_port
> ```
> 
> It's not necessary :) the rest lgtm!

We decided to keep `urllib` to avoid external dependency, but yes I agree request is easier and more readable

File: tests/integration/issues/github_976/test_topk_rest_api.py [hunk 2]
```diff
@@ -1,22 +1,25 @@
-import pytest
 import json
+import random
```
Comment (JoanFM) [inline tests/integration/issues/github_976/test_topk_rest_api.py:2]: remove this import

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by bhavsarpratik** [comment]
I like requests too but I don't see it in our `extra-requirements.txt`. So we won't be able to use it.

**Review by bwanglzu — APPROVED** [review]
LGTM

**Pull request merged**